tracking-snippet.php

```diff
@@ -4,7 +4,10 @@
 tap('detect');
 
 <?php
+$external_id_arg = apply_filters('tapfiliate_snippet_external_id_arg', $external_id_arg);
```

Comment:
This should happen in the controller code, in Tapfiliate.php `tapfiliate_output_inline_code`.